README.md

```diff
@@ -209,19 +226,19 @@ You need to use the eval.py script.
 
 You can generate more images from an existing checkpoint using:
 ```
-python eval.py visualization -n $modelName -m $modelType
```

Comment:
there was a confusion between model name, model type and run name --- I propose this version.